### DISCUSSION TITLE
Adding the German BSI-IT-Grundschutz

### DISCUSSION THREAD

**Discussion by 42mst** [intuitem/ciso-assistant-community#240] [Ideas]
To expand even further I think it would be a good idea to include this as well.

**Comment by eric-intuitem** [comment]
@42mst we will look at that, thanks for the suggestion.

**Reply by 42mst** [reply]
@eric-intuitem 
I think currently there is no latest version of the IT-Grundschutz available. I´m not sure about your data formatting but I could provide the latest version as an xlsx or yaml. You could then adjust it as needed.

**Reply by ab-smith** [reply]
hey @42mst that will be very helpful indeed, thank you in advance 👍

**Reply by 42mst** [reply]
@ab-smith @eric-intuitem 
this is probably not the model format but will provide you everything you need.

@ab-smith Did you had the time to think about the other comment I made in the issue #185 ?

[IT-GS-Kompendium_ed2023.xlsx](https://github.com/intuitem/ciso-assistant-community/files/15076068/IT-GS-Kompendium_ed2023.xlsx)
[IT-GS-Kompendium_ed2023.json](https://github.com/intuitem/ciso-assistant-community/files/15076070/IT-GS-Kompendium_ed2023.json)

**Reply by lfrancke** [reply]
IT-Grundschutz is what brought me to CISO Assistant.
Seconding the request. :heart: 

There will be no 2024 version of IT-Grundschutz and the latest english version is from 2022 I believe.
Unfortunately, there is also no machine-readable version of the controls but there are various tools to extract them from the PDF files. @42mst has provided one such extract.

I'm also happy to help out massaging data to conform to whatever format you need.

**Reply by 42mst** [reply]
@lfrancke @ab-smith 
I also have the IT-Grundschutz in an easier format with only the most relevant information. If this would be easier to format than just let me know.

**Reply by 42mst** [reply]
@lfrancke @ab-smith 
And yes, having the IT-Grundschutz in CISO assistant will open a lot more opportunities and will even provide more attention in the DACH-Region

**Comment by ab-smith** [comment]
hey thanks for sharing the files and sorry I've missed your message,
I've put some comments on the issue, let me know what you think

**Reply by ab-smith** [reply]
Let's focus on this one and track our interaction on #357 I've added something there @42mst
